src/components/LoginSignUp/AuthenticationService.js

```diff
@@ -82,10 +82,11 @@ const getAnonymousJWTToken = () => {
  * @returns {Promise<string|*>}
  */
 const getJWTToken = async ( allowAnonymousJWTToken: boolean = false ): Promise<?string> => {
-  let jwtToken = await RNSInfo.getItem( "jwtToken", {} );
-  let jwtTokenExpiration = await RNSInfo.getItem( "jwtTokenExpiration", {} );
-  if ( jwtTokenExpiration ) {
-    jwtTokenExpiration = parseInt( jwtTokenExpiration, 10 );
+  let jwtToken: string | void = await RNSInfo.getItem( "jwtToken", {} );
+  const jwtTokenExpirationString: string | void = await RNSInfo.getItem( "jwtTokenExpiration", {} );
+  let jwtTokenExpiration: number | void;
+  if ( jwtTokenExpirationString ) {
+    jwtTokenExpiration = parseInt( jwtTokenExpirationString, 10 );
```

Comment:
Just stylistic, but why not something like

```javascript
const jwtExpiraton: number | void = parseInt( await RNSInfo.getItem( "jwtTokenExpiration", {} ), 10 )
```

I *think* `parseInt` is pretty robust against various kinds of null-ish values it might receive.

Reply:
`parseInt(null)` returns `NaN`. So if `( Date.now() - NaN ) / 1000 > JWT_TOKEN_EXPIRATION_MINS * 60` is okay with you, then I can change it

Reply:
I'm pretty sure that would evaluate in the way we want it to, so yeah, I'd be fine with that. No strong feeling about it here, so feel free to leave it.